### PR TITLE
Java solve for triangles

### PR DESCRIPTION
**Pull request by MrinallU**

### CODE REVIEW

File: solutions/USACO feb20-triangles.mdx
Comment (darren-yao) [inline solutions/USACO feb20-triangles.mdx:29]: it's spelled vertices

Comment (MrinallU) [reply solutions/USACO feb20-triangles.mdx:29]: Sorry about that!

File: solutions/USACO feb20-triangles.mdx
Comment (bqi343) [inline solutions/USACO feb20-triangles.mdx:32]: what's up with the spacing o_O

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/USACO feb20-triangles.mdx
Comment (MrinallU) [reply solutions/USACO feb20-triangles.mdx:32]: Oh, Idk why I did that.

File: solutions/USACO feb20-triangles.mdx
Comment (dongliu0426) [inline solutions/USACO feb20-triangles.mdx:8]: the link you put here is the problem statement not the editorial

File: solutions/USACO feb20-triangles.mdx
Comment (dongliu0426) [inline solutions/USACO feb20-triangles.mdx:13]: maybe make a note that the brute force takes N^3

File: solutions/USACO feb20-triangles.mdx
Comment (dongliu0426) [inline solutions/USACO feb20-triangles.mdx:44]:
```suggestion
                    int area = Math.abs((a_x  * (b_y - c_y)) + (b_x * (c_y - a_y)) + (c_x * (a_y - b_y ))); // Calculate the area of the three triangles
```

**Review by dongliu0426 — CHANGES_REQUESTED** [review]

File: solutions/USACO feb20-triangles.mdx
Comment (MrinallU) [reply solutions/USACO feb20-triangles.mdx:13]: I just copied and pasted the editorial from the original solution page on the guide but I will add that as well if you would like.

Comment (MrinallU) [reply solutions/USACO feb20-triangles.mdx]: Done

File: solutions/USACO feb20-triangles.mdx
Comment (dongliu0426) [inline solutions/USACO feb20-triangles.mdx:3]: uh why are there two `2020`'s

File: solutions/USACO feb20-triangles.mdx
Comment (dongliu0426) [inline solutions/USACO feb20-triangles.mdx:12]:
```suggestion
Brute force all possible right triangles by looping over all triples of points and checking whether they form a right triangle. If so, compute the area and find the maximum over all right triangles all in $\mathcal{O}(N^3)$ time.
```

**Review by dongliu0426 — CHANGES_REQUESTED** [review]

File: solutions/USACO feb20-triangles.mdx
Comment (dongliu0426) [reply solutions/USACO feb20-triangles.mdx:3]:
```suggestion
source: USACO Bronze 2020 February
```

**Review by dongliu0426 — APPROVED** [review]
looks good. just wait for someone from the core team to review this.

**Comment by MrinallU** [comment]
@dongliu0426 Sorry for the careless mistakes by the way, I made sure to take care of those in the second pull request I posted.

**Comment by MrinallU** [comment]
@bqi343 I think this is ready to merge?

**Comment by bqi343** [comment]
I am not the one in charge of reviewing this lol @darren-yao?

**Review by darren-yao — APPROVED** [review]
approved

**Pull request merged**
